### ISSUE TITLE
Missing comma(s) causes an error

### ISSUE THREAD

**Issue by genos** [PennyLaneAI/pennylane-rigetti#119]
# Intro

First off, thanks _very_ much for this great plugin, it's really exciting.

I've run into a bug when trying to use the latest versions. Because [`requirements.txt`](https://github.com/PennyLaneAI/pennylane-rigetti/blob/master/requirements.txt#L3), [`setup.py`](https://github.com/PennyLaneAI/pennylane-rigetti/blob/master/setup.py#L10), or both are missing a comma between the lower and upper bounds in `qcs-api-client>=0.20.13<0.22.0`, I get an `AttributeError` and a couple of other friends. I think putting that comma in should fix the issue; I'll put up a PR shortly.

Thanks again!

# Reproducing the error

## Setup
```shell
~/tmp/reprex $ python -mvenv venv 
~/tmp/reprex $ source venv/bin/activate
(venv) ~/tmp/reprex $ pip install -U -qqq pip setuptools wheel Pennylane Pennylane-Rigetti
(venv) ~/tmp/reprex $ pip freeze
anyio==3.6.2
appdirs==1.4.4
attrs==21.4.0
autograd==1.5
autoray==0.6.0
cachetools==5.3.0
certifi==2022.12.7
charset-normalizer==3.0.1
decorator==5.1.1
future==0.18.3
h11==0.14.0
httpcore==0.16.3
httpx==0.23.3
idna==3.4
iso8601==1.1.0
lark==0.11.3
msgpack==0.6.2
networkx==2.8.8
ninja==1.11.1
numpy==1.23.5
PennyLane==0.28.0
PennyLane-Lightning==0.28.2
PennyLane-Rigetti==0.27.0
py==1.11.0
pydantic==1.10.4
PyJWT==2.6.0
pyquil==3.3.3
python-dateutil==2.8.2
python-rapidjson==1.9
pyzmq==25.0.0
qcs-api-client==0.21.3
requests==2.28.2
retry==0.9.2
retrying==1.3.4
retworkx==0.12.1
rfc3339==6.2
rfc3986==1.5.0
rpcq==3.10.0
ruamel.yaml==0.17.21
ruamel.yaml.clib==0.2.7
rustworkx==0.12.1
scipy==1.10.0
semantic-version==2.10.0
six==1.16.0
sniffio==1.3.0
toml==0.10.2
types-python-dateutil==2.8.19.6
types-retry==0.9.9.1
typing_extensions==4.4.0
urllib3==1.26.14
```

## Error

```python
>>> import pennylane as qml
>>> qml.device("rigetti.qpu", device="Aspen-M-2")
Traceback (most recent call last):
  File "/Users/genos/tmp/reprex/venv/lib/python3.10/site-packages/pkg_resources/__init__.py", line 3030, in _dep_map
    return self.__dep_map
  File "/Users/genos/tmp/reprex/venv/lib/python3.10/site-packages/pkg_resources/__init__.py", line 2826, in __getattr__
    raise AttributeError(attr)
AttributeError: _DistInfoDistribution__dep_map

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/genos/tmp/reprex/venv/lib/python3.10/site-packages/pkg_resources/_vendor/packaging/requirements.py", line 35, in __init__
    parsed = parse_requirement(requirement_string)
  File "/Users/genos/tmp/reprex/venv/lib/python3.10/site-packages/pkg_resources/_vendor/packaging/_parser.py", line 64, in parse_requirement
    return _parse_requirement(Tokenizer(source, rules=DEFAULT_RULES))
  File "/Users/genos/tmp/reprex/venv/lib/python3.10/site-packages/pkg_resources/_vendor/packaging/_parser.py", line 82, in _parse_requirement
    url, specifier, marker = _parse_requirement_details(tokenizer)
  File "/Users/genos/tmp/reprex/venv/lib/python3.10/site-packages/pkg_resources/_vendor/packaging/_parser.py", line 120, in _parse_requirement_details
    specifier = _parse_specifier(tokenizer)
  File "/Users/genos/tmp/reprex/venv/lib/python3.10/site-packages/pkg_resources/_vendor/packaging/_parser.py", line 206, in _parse_specifier
    with tokenizer.enclosing_tokens("LEFT_PARENTHESIS", "RIGHT_PARENTHESIS"):
  File "/Users/genos/.pyenv/versions/3.10.7/lib/python3.10/contextlib.py", line 142, in __exit__
    next(self.gen)
  File "/Users/genos/tmp/reprex/venv/lib/python3.10/site-packages/pkg_resources/_vendor/packaging/_tokenizer.py", line 183, in enclosing_tokens
    self.raise_syntax_error(
  File "/Users/genos/tmp/reprex/venv/lib/python3.10/site-packages/pkg_resources/_vendor/packaging/_tokenizer.py", line 163, in raise_syntax_error
    raise ParserSyntaxError(
pkg_resources.extern.packaging._tokenizer.ParserSyntaxError: Expected closing RIGHT_PARENTHESIS
    qcs-api-client (>=0.20.13<0.22.0)
                   ~~~~~~~~~~^

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/genos/tmp/reprex/venv/lib/python3.10/site-packages/pennylane/__init__.py", line 316, in device
    plugin_device_class = plugin_devices[name].load()
  File "/Users/genos/tmp/reprex/venv/lib/python3.10/site-packages/pkg_resources/__init__.py", line 2467, in load
    self.require(*args, **kwargs)
  File "/Users/genos/tmp/reprex/venv/lib/python3.10/site-packages/pkg_resources/__init__.py", line 2489, in require
    reqs = self.dist.requires(self.extras)
  File "/Users/genos/tmp/reprex/venv/lib/python3.10/site-packages/pkg_resources/__init__.py", line 2746, in requires
    dm = self._dep_map
  File "/Users/genos/tmp/reprex/venv/lib/python3.10/site-packages/pkg_resources/__init__.py", line 3032, in _dep_map
    self.__dep_map = self._compute_dependencies()
  File "/Users/genos/tmp/reprex/venv/lib/python3.10/site-packages/pkg_resources/__init__.py", line 3042, in _compute_dependencies
    reqs.extend(parse_requirements(req))
  File "/Users/genos/tmp/reprex/venv/lib/python3.10/site-packages/pkg_resources/__init__.py", line 3095, in __init__
    super(Requirement, self).__init__(requirement_string)
  File "/Users/genos/tmp/reprex/venv/lib/python3.10/site-packages/pkg_resources/_vendor/packaging/requirements.py", line 37, in __init__
    raise InvalidRequirement(str(e)) from e
pkg_resources.extern.packaging.requirements.InvalidRequirement: Expected closing RIGHT_PARENTHESIS
    qcs-api-client (>=0.20.13<0.22.0)
                   ~~~~~~~~~~^
```

**Comment by isaacdevlugt** [comment]
Hey @genos! Looks like this is getting addressed in the PR you made. Any other concerns here?

**Comment by genos** [comment]
Hi @isaacdevlugt! Thanks for checking in. The PR should address this, though there's been a bit of churn there (my fault): in addressing how dependencies are specified, I've also touched the CI setup—and as an outsider without :octocat: permissions to take over and run things (and I shouldn't have them, I just showed up to play in someone else's backyard!), it's a little slow going verifying the PR's structural changes and making sure things work.

**Comment by isaacdevlugt** [comment]
Yep, we do pride ourselves on rigorous checks and balances! Feel free to close this issue when you see fit, or notify me if you're experiencing any more difficulties 😃.

**Comment by genos** [comment]
Thanks! If possible, could you take a look at the PR and, if it’s to your liking, allow the CI checks to run? It’s awaiting approval from a project maintainer to start CI (I guess to guard against misuse).

**Comment by genos** [comment]
Closed by #120

thanks all!
